cob_hardware_emulation/scripts/emulation_clock.py

```diff
@@ -7,16 +7,16 @@
 class EmulationClock(object):
     def __init__(self):
         self.t = time.time()
-        self.dt = 0.01
+        self.dt_ms = rospy.get_param('/emulation_dt_ms', 10)
         self.time_factor = rospy.get_param('/emulation_time_factor', 1.0)
         if not self.time_factor > 0.0:
             rospy.logerr("emulation_time_factor must be >0.0, but is {}. exiting...".format(self.time_factor))
             exit(-1)
         self.pub = rospy.Publisher('/clock', Clock, queue_size=1)
-        rospy.Timer(rospy.Duration(self.dt), self.timer_cb)
+        rospy.Timer(rospy.Duration(self.dt_ms/1000.0), self.timer_cb)
```

Comment:
The official docs say the 'secs' arg or `rospy.Duration` should be an int, so no need for the `.0` at the end of `1000.0` if we're strict. Of course, everyone uses it a float which work in Python.